pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -129,3 +131,150 @@ func (nameScoping *DefaultNameScopingImpl) Handle(command utils.CommandEnum, clu
 	}
 	return nil
 }
+
+func CheckContainerReferences(cluster cluster.Cluster, tenantId string, containerConfig *dockerclient.ContainerConfig) error {
+	log.Debugf("CheckContainerReferences containerConfig: %+v", containerConfig)
+	// create arrays of container references to pass to getIDsFromContainerReferences
+	// create volumesFrom array
+	volumesFrom := containerConfig.HostConfig.VolumesFrom
+
+	// create affinity array
+	affinityRef := make([]string, 0)
+	env := containerConfig.Env
+	var affinityIndex int
+	affinityLabelFirst := false
+	for i, e := range env {
```

Comment:
you break from the loop on the first affinity , can one specify more then a single affinity ?